From 95a99d622a18b5d7c1184ea1ea348e9ba6a0c8eb Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Mon, 17 May 2021 16:31:36 +0800
Subject: [PATCH 06/13] dts: s32g27a: pfe: move reserve memory address to leave
 enough space for kernel image

For normal kernel image, it is acceptable if pfe reserved-memory starts
from 83000000, but for bts or kts v5.10 kernel image, due to the kernel
image much larger than normal, so move pfe reserved-memory address to
8f000000 to avoid intersect with kernel image, otherwise, the pfe will
failed when try to reserve memory, and can not work then.

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
index 2bbdb2eed652..f07b5ba28cc4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
@@ -34,41 +34,41 @@ reserved-memory {
 		#size-cells = <2>;
 		ranges;
 
-		pfe_reserved_bmu2: pfebufs@83000000 {
+		pfe_reserved_bmu2: pfebufs@8f000000 {
 			compatible = "fsl,pfe-bmu2-pool";
 			/* 2 MB */
-			reg = <0 0x83000000 0 0x200000>;
+			reg = <0 0x8f000000 0 0x200000>;
 			no-map;
 			status = "okay";
 		};
 
-		pfe_reserved: pfebufs@83200000 {
+		pfe_reserved: pfebufs@8f200000 {
 			compatible = "shared-dma-pool";
 			/* 4 MB - 128 KB */
-			reg = <0 0x83200000 0 0x3e0000>;
+			reg = <0 0x8f200000 0 0x3e0000>;
 			no-map;
 			status = "okay";
 		};
 
-		pfe_reserved_bdr: pfebufs@83bf0000 {
+		pfe_reserved_bdr: pfebufs@8f5e0000 {
 			compatible = "fsl,pfe-bdr-pool";
 			/* 128 KB */
-			reg = <0 0x835e0000 0 0x20000>;
+			reg = <0 0x8f5e0000 0 0x20000>;
 			status = "okay";
 		};
 
-		pfe_reserved_slave: pfebufs@83600000 {
+		pfe_reserved_slave: pfebufs@8f600000 {
 			compatible = "shared-dma-pool";
 			/* 2 MB - 128 KB */
-			reg = <0 0x83600000 0 0x1e0000>;
+			reg = <0 0x8f600000 0 0x1e0000>;
 			no-map;
 			status = "disabled";
 		};
 
-		pfe_reserved_bdr_slave: pfebufs@837e0000 {
+		pfe_reserved_bdr_slave: pfebufs@8f7e0000 {
 			compatible = "fsl,pfe-bdr-pool";
 			/* 128 KB */
-			reg = <0 0x837e0000 0 0x20000>;
+			reg = <0 0x8f7e0000 0 0x20000>;
 			status = "disabled";
 		};
 		/* ends 0x83800000 */
-- 
2.25.1

